### PR TITLE
Added wavpack audio encoder

### PR DESCRIPTION
**Pull request by Kleinrotti**

### CODE REVIEW

File: Core/resources/encoders/wavpack.json [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      "class": "standard",
+      "properties": [
+        {
+          "id": "wavpack.standard.sampleFormat",
+          "parameter": "_sampleFormat",
+          "forced": true,
+          "control": {
+            "type": "combobox",
+            "selectedIndex": 2,
+            "items": [
```
Comment (Vouk) [inline Core/resources/encoders/wavpack.json]: Please reorder to: 8, 16, 32, fltp -> Looks nicer to the user

File: Core/resources/translations/en.json [hunk 2]
```diff
@@ -2003,6 +2003,28 @@
     "pcm.standard.sampleFormat._item_0": "PCM 16 bit",
     "pcm.standard.sampleFormat._item_1": "PCM 24 bit",
     "pcm.standard.sampleFormat._item_2": "PCM 32 bit",
+    "wavpack.standard": "Standard",
+    "wavpack.standard.sampleFormat._label": "Sample format",
+    "wavpack.standard.sampleFormat._description": "Constant encoding bitrate.",
```
Comment (Vouk) [inline Core/resources/translations/en.json]: Fix description text

Comment (Kleinrotti) [reply Core/resources/translations/en.json]: Now it should be fine.

File: Core/resources/encoders/wavpack.json (hunk 1 above)
Comment (Kleinrotti) [reply Core/resources/encoders/wavpack.json]: Should be fine now.

**Pull request merged**